File: src/main/java/com/procurement/point/config/properties/OCDSProperties.java
package com.procurement.point.config.properties;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Getter
@Setter
@Component
@ConfigurationProperties(prefix = "ocds")
public class OCDSProperties {

    private String[] extensions;

    private String path;

    private String version;

    private String license;

    private String publicationPolicy;

    private String publisherName;

    private String publisherScheme;

    private String publisherUid;

    private String publisherUri;

}

File: src/main/kotlin/com/procurement/point/config/OCDSProperties.kt
package com.procurement.point.config

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "ocds")
data class OCDSProperties(

        var path: String?,

        var publisherScheme: String?,

        var publisherUid: String?,

        var defLimit: Int?,

        var maxLimit: Int?
)
